### DISCUSSION TITLE
Names of imports are hidden

### DISCUSSION THREAD

**Discussion by ChrisAmelia** [mfussenegger/nvim-jdtls#474] [Q&A] [answered]
This has happened after updating my plugins and jdtls. Not sure if this is related to `nvim-jdtls` or `nvim-cmp`.
When looking for a package, the name of it is hidden, see this screenshot

![image](https://user-images.githubusercontent.com/16366403/234528963-5eb130ab-74fc-4ff6-b02c-8511653e912c.png)

It used to be display as:

```
List java.util.List
List atlr.collections.List
List com.github.andrewoma.dexx.collection.List
```

So now, I have to check manually for the package I want.

The behaviour is more general, methods' signature are also shortened:

![image](https://user-images.githubusercontent.com/16366403/234532605-3feeecce-4fb9-4830-9ece-7ef54d5c8a3d.png)

Expected behaviour: methods' signature are also displayed without the need to put the cursor on it:

```
indexOf(String str, int fromIndex)
indexOf(int ch, int fromIndex)
indexOf(String str)
idnexOf(int ch)
```

**Comment by mfussenegger** [comment, marked as answer]
See https://github.com/mfussenegger/nvim-jdtls/issues/467#issuecomment-1512453415 & https://github.com/mfussenegger/nvim-jdtls/issues/467#issuecomment-1513362893
